driver/examples/src/test/scala/org/virtuslab/ideprobe/ModuleTestRun.scala

```diff
@@ -0,0 +1,24 @@
+package org.virtuslab.ideprobe
+
+import org.junit.{Assert, Test}
+import org.virtuslab.ideprobe.protocol.{JUnitRunConfiguration, ModuleRef}
+
+class ModuleTestRun extends ModuleTest {
```

Comment:
this also can become a single method in the `ModuleTest` class called `runsAllTestsInModule`